src/cascadia/TerminalSettingsEditor/MainPage.cpp

```diff
@@ -391,9 +392,13 @@ namespace winrt::Microsoft::Terminal::Settings::Editor::implementation
         else if (clickedItemTag == globalProfileTag)
         {
             auto profileVM{ _viewModelForProfile(_settingsClone.ProfileDefaults(), _settingsClone) };
+            profileVM.DefaultAppearance().SchemesPageVM(_colorSchemesPageVM);
+            if (profileVM.UnfocusedAppearance())
+            {
+                profileVM.UnfocusedAppearance().SchemesPageVM(_colorSchemesPageVM);
+            }

```

Comment:
This should go into a helper since it is used in a few places

Reply:
Maybe you could do something like:
```cpp
profileVM.ApplySchemesPageVM(_colorSchemesPageVM);
```

and the ProfileVM applies it to all appearances?

Reply:
Just called it `SetupAppearances` because we also need to pass in the window root